cmd/cli/commands/infrastructure.go

```diff
@@ -106,13 +107,13 @@ var deleteInfraCmd = &cobra.Command{
 	Use:   "delete",
 	Short: "Delete infrastructure",
 	RunE: func(cmd *cobra.Command, _ []string) error {
-		jsonFile, _ := cmd.Flags().GetString("file")
-		if err := validateFilePath(jsonFile); err != nil {
```

Comment:
is `validateFilePath` not used now? can that function be deleted?

Reply:
it is still used by some other commands that are outside of the scope of this PR. let's create a followup to delete it as its function could be achieved by `os.ReadFile` which throws an error if the filepath is not right or does not exist.
@coderabbitai, would you please create an issue for it?